sass/components/_post.scss

```diff
@@ -2006,6 +2006,30 @@
     animation-fill-mode: forwards;
 }
 
+// Handle fade out effect for permalink highlightning
+.app__body .post-list__table .post:not(.current--user).post--highlight .post-collapse__gradient,
+.app__body .post-list__table .post.post--compact.post--highlight .post-collapse__gradient {
+    background:linear-gradient(rgba(var(--mention-highlight-bg-mixed-rgb), 0), rgba(var(--mention-highlight-bg-mixed-rgb), 1));
+}
+
+.app__body .post-list__table .post:not(current--user).post--highlight:not(.post--compact) .post-collapse>.post-collapse__show-more
```

Comment:
Is the `.post-collapse>` part of the rule necessary? There's definitely other parts that aren't necessary, but that will keep these consistent with the other rules that look at the post and then the show-more.

Reply:
Most likely no, but I'll re-check.